text/0034-telemetry-policies.md

```diff
@@ -522,6 +526,8 @@ This property allows us to gather essential information to better understand on
 |---------------|-------------|---------|
 | user_agent    | Represents the user-agent encountered on this call. | `["Meilisearch Ruby (v2.1)", "Ruby (3.0)"]` |
 | faceting.max_values_per_facet | The defined value for `maxValuesPerFacet` property | `100` |
+| faceting.sort_facet_values_by_star_count | Whether the user set all fields to be sort by count | `true` |
```

Comment:
Why is this called `faceting.sort_facet_values_by_star_count` instead of just `faceting.sort_facet_values_by_count`?

Reply:
Hey @maryamsulemani97

We want to track if the default `*` is expressly set to `count` to re-evaluate the default value in the future (today it's `alpha`). tbh we didn't spent too much time thinking about the name, that's a telemetry point nothing critical here 😇